.azure-pipelines/generation-templates/java-beta.yml

```diff
@@ -0,0 +1,47 @@
+steps:
```

Comment:
instead of relying on the variables directly, we could rely on template parameters. That would allow us to use the same template for v1 and beta. The parameters can be initialized with different variables where they are referenced.